### DISCUSSION TITLE
How to impement tooltip in fullcalendar with Alpinejs?

### DISCUSSION THREAD

**Discussion by sergeynilov** [alpinejs/alpine#1238] [1. Help]
Hello,

I added fullcalendar v4.3.1 to Laravel 8 /  tailwindcss 2 / Alpinejs 2.8 app 
and it works, but I encountered a question how to impement tooltips with keeping
mouve over date cell? With jquery support I made with tooltip libarary :
```
    events: function(info, successCallback, failureCallback) {    //get data from db for selected dates
        $(eventInfo.el).tooltip({
            title: adHtml,
            html: true,
        })

```
I found example with tailwindcss tooltip impementation when button is on the form :
https://codepen.io/t7team/pen/XWdyVyB

But with fullcalendar it seems tricky...

How it can be done ?

Thanks!

**Comment by KevinBatdorf** [comment]
Maybe you can try Tippyjs? https://atomiks.github.io/tippyjs/

**Reply by sergeynilov** [reply]
Thanks! It helped.
